code/modules/client/verbs/ooc.dm

```diff
@@ -93,10 +93,21 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 				else
 					to_chat(C, "[badge_data]<span class='ooc'><span class='prefix'>OOC:</span> <EM>[keyname]:</EM> <span class='message linkify'>[msg]</span></span>")
 	// beestation, send to discord
-	if(holder?.fakekey)
-		discordsendmsg("ooc", "**[holder.fakekey]:** [msg]")
-	else
-		discordsendmsg("ooc", "**[key]:** [msg]")
+	send_chat_to_discord("ooc_chat", holder?.fakekey || key, raw_msg)
```

Comment:
use DEFINES for these strings

Reply:
do you mean `ooc_chat` thingies?

Reply:
Yes, they're magic strings currently which is bad

Reply:
done